frontend/packages/dev-console/src/components/GuidedTourTile.tsx

```diff
@@ -19,33 +31,37 @@ const GuidedTourItem: React.FC<GuidedTourTileProps> = ({ tours }) => {
     e.preventDefault();
   };
   const orderedTours = tours.length > 3 ? tours.slice(0, 3) : tours;
-  const tileDescription = (
-    <>
-      {orderedTours.map((tour) => (
-        <div key={tour.name}>
-          <Button variant="link">{tour.name}</Button>
-        </div>
-      ))}
-    </>
-  );
-  const tileFooter = (
-    <>
-      <Divider component="div" />
-      <ArrowRightIcon />
-      <Button variant="link" onClick={(e: React.SyntheticEvent) => navigateTo(e, '/tours/')}>
-        See all guided tours
-      </Button>
-    </>
-  );
   return (
-    <GalleryItem>
-      <CatalogTile
-        data-test-id="guided-tour"
-        className="odc-empty-state__tile"
-        title={<h2>Guided Tours</h2>}
-        description={tileDescription}
-        footer={tileFooter}
-      />
+    <GalleryItem className="odc-guidedtour-tile">
+      <Card className="odc-guidedtour-tile__card">
+        <CardHeader>
+          <Flex className="odc-guidedtour-tile__title">
+            <FlexItem>
+              <Text component={TextVariants.h3}> Guided Tours</Text>
```

Comment:
Use `CardTitle`. See PF examples: https://patternfly-react.surge.sh/documentation/react/components/card

Reply:
CardTitle seems to be unavailable with current PF

Reply:
The PF v4 PR is on its way. Maybe just wait for it to merge otherwise we'll have to change this again as soon as it's in.
Or look at the previous release docs and find common components between the two that works.